frontend/javascripts/oxalis/view/right-border-tabs/segments_tab/segments_view.tsx

```diff
@@ -932,7 +924,7 @@ class SegmentsView extends React.Component<Props, State> {
       disabled: isEditingDisabled,
       icon: (
         <i
-          className="fas fa-eye-dropper fa-sm fa-icon"
+          className="fas fa-eye-dropper fa-sm fa-icon fa-fw "
```

Comment:
```suggestion
          className="fas fa-eye-dropper fa-sm fa-icon fa-fw"
```

what does `fa-fw` do?

Reply:
it makes all the icons have the same width (https://fontawesome.com/v5/docs/web/style/fixed-width)

Reply:
cool, I didn't know about that :)